test/e2e-common.sh

```diff
@@ -124,7 +124,7 @@ function run_yaml_tests() {
 
 function install_pipeline_crd() {
   echo ">> Deploying Tekton Pipelines"
-  ko apply -f config/ || fail_test "Build pipeline installation failed"
+  kubectl apply -f config/ || fail_test "Build pipeline installation failed"
```

Comment:
should be `kubectl apply -f https://github.com/tektoncd/pipeline/releases/download/v0.4.0/release.yaml` :angel:

Reply:
Copy paste error :)